src/DatabaseSeeker.php

```diff
@@ -104,6 +101,33 @@ private function performSearch(Builder $builder, array $keywords, int $page, ?in
         return new SearchResult($builder, $results, $totalHits);
     }
 
+    /**
+     * Add wildcards to the given keywords if configured.
+     *
+     * @param string[] $keywords
+     */
+    private function addWildcards(array $keywords): array
+    {
+        // Add a wildcard to each search token if it is configured.
+        if ($this->searchConfiguration->allTokensShouldUseWildcard()) {
+            $pattern = $this->searchConfiguration->allTokensShouldUseWildcard() === 'both' ? '%%%s%%' : '%s%%';
+
+            $keywords = array_map(
```

Comment:
I think we should `return` here, otherwise configuring both, `wildcard_all_tokens` and `wildcard_last_token`, will cause double wildcards (probably not harmful, but also not very useful).

Reply:
I think it is an advantage that both settings are possible in parallel. Because with an autosuggest search you might want to get results regardless of the minimum length for the last token. I even added a test for this since this is intended behavior imho. What do you think?